hal/halx86/generic/up.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Core source file for Uniprocessor (UP) alternative functions
+ * COPYRIGHT:   Copyright 2021 Justin Miller <justinmiller100@gmail.com>
```

Comment:
```suggestion
 * COPYRIGHT:   Copyright 2023 Justin Miller <justinmiller100@gmail.com>
```
The file was newly added in 2023.